scheduler/src/cook/kubernetes/controller.clj

```diff
@@ -142,43 +144,51 @@
                                          (.getName container-status))))
        first))
 
+(defn synthetic-pod-job-uuid
```

Comment:
Should be synthetic-pod->job-uuid
Also V1Pod annotation on pod.

Reply:
Good call, will do, thanks

Reply:
Renamed and added type annotation